cluster/gce/util.sh

```diff
@@ -3067,7 +3111,7 @@ function create-master() {
   MASTER_RESERVED_IP=$(gcloud compute addresses describe "${MASTER_NAME}-ip" \
     --project "${PROJECT}" --region "${REGION}" -q --format='value(address)')
 
-  echo ${MASTER_RESERVED_IP} > /tmp/master_reserved_ip.txt
+  echo ${MASTER_RESERVED_IP} > ${KUBE_TEMP}/master_reserved_ip.txt
```

Comment:
This can be read from kubeconfig as well. I cleaned up a lot of this stuff. Why do we need this?

Reply:
same as above, it is a typical way to get subprocess returns in bash.